### PR TITLE
chore(compute): add pagination samples

### PR DESCRIPTION
**Pull request by yuryu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-cloud-ruby/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
- [ ] Follow the instructions in [CONTRIBUTING](CONTRIBUTING.md). Most importantly, ensure the tests and linter pass by running `bundle exec rake ci` in the gem subdirectory.
- [ ] Update code documentation if necessary.

Add pagination samples for the new compute api.

closes: #<issue_number_goes_here>

### CODE REVIEW

File: google-cloud-compute-v1/samples/pagination.rb
Comment (rsamborski) [inline google-cloud-compute-v1/samples/pagination.rb:34]: Based on arguments provided by @amanda-tarafa in [this comment](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1445/files#r687921807), please change to 100.

**Review by rsamborski — APPROVED** [review]
Small nit regarding number of MaxValues. LGTM besides that.

**Comment by yuryu** [comment]
Addressed @rsamborski's comment. Need review from @googleapis/yoshi-ruby before merging.

File: google-cloud-compute-v1/samples/acceptance/pagination_test.rb
Comment (dazuma) [inline google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: I assume successful test runs would require credentials for this project? Our CI is run against the project `helical-zone-771` (as in, the CI job has credentials for that project) so we probably need to use that project and set up test fixtures in that project instead. So the fact that CI appears to be running tests and succeeding, is puzzling to me. Can you explain what's going on?

Comment (rsamborski) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: windows-sql-cloud is a special project which stores public SQL Server images. It should be available to any GCP project. PTAL here: https://cloud.google.com/compute/docs/images/os-details#sql_server

Comment (yuryu) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: Yes, it should remain accessible and tests continue to pass. I used it following the python samples.

Comment (dazuma) [reply google-cloud-compute-v1/samples/acceptance/pagination_test.rb:20]: ok.

**Review by dazuma — APPROVED** [review]

**Comment by yuryu** [comment]
So again, I need someone to merge this 🙏

**Pull request merged**
